velox/type/Type.cpp

```diff
@@ -1032,6 +1033,33 @@ int32_t DateType::toDays(const char* in, size_t len) const {
   return util::fromDateString(in, len);
 }
 
+std::string TimeMillisType::valueToString(
+    int64_t millis,
+    const date::time_zone* zone) {
+  int64_t millisOfDay = velox::TimeUtil::getMillisOfDay(millis);
+  if (zone != nullptr) {
+    millisOfDay = velox::TimeUtil::toTimezone(millisOfDay, *zone);
+  }
+
+  auto seconds = millisOfDay / velox::TimeUtil::kMillisecondsInSecond;
+  auto milliseconds = millisOfDay % velox::TimeUtil::kMillisecondsInSecond;
+  std::tm tmValue{};
+  VELOX_USER_CHECK_NOT_NULL(
+      gmtime_r((const time_t*)&seconds, &tmValue),
```

Comment:
No `gmtime_r` in Velox; use `epochToUtc` instead: https://github.com/facebookincubator/velox/blob/a7ba081c6c62cab3ba4072f18e3dd37e31200458/velox/type/Timestamp.h#L50

Reply:
@Yuhta Fixed, Thanks.